<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>421</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>920</width>
    <height>421</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>920</width>
    <height>421</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dashboard (Server)</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-10</y>
      <width>981</width>
      <height>491</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>dash.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="speed">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
						color: white;
						font-size: 50px;
					</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:38pt;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
						color: white;
						font-size: 10px;
					</string>
    </property>
    <property name="text">
     <string>km/h</string>
    </property>
   </widget>
   <widget class="QLabel" name="degree">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>210</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
						color: white;
						font-size: 50px;
					</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>471</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
						#centralWidget {
						border-image: url(:/res/background.jpg) 0 0 0 0 stretch stretch;
						}
					</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Pictures/2022-06-25 at 15-10-25.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="larrow_b">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Oparrow.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="light_b">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>280</y>
      <width>49</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>lightOn.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="fuel_b">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>lowfuel.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="fix_b">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>280</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>fix.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="battery_b">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>280</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>batterydown.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="opendoor_b">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>280</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>opendoor.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="rarrow_b">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>130</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>arrow.png</pixmap>
    </property>
   </widget>
   <widget class="QCheckBox" name="larrow_c">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>461</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="rarrow_c">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>90</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fix_c">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>340</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="battery_c">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>340</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="light_c">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>340</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="opendoor_c">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>340</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fuel_c">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>240</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>350</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>8</number>
    </property>
    <property name="pageStep">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="output">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white</string>
    </property>
    <property name="text">
     <string>output</string>
    </property>
   </widget>
   <zorder>larrow_c</zorder>
   <zorder>rarrow_c</zorder>
   <zorder>fix_c</zorder>
   <zorder>horizontalSlider_2</zorder>
   <zorder>battery_c</zorder>
   <zorder>light_c</zorder>
   <zorder>opendoor_c</zorder>
   <zorder>fuel_c</zorder>
   <zorder>horizontalSlider</zorder>
   <zorder>label</zorder>
   <zorder>speed</zorder>
   <zorder>label_3</zorder>
   <zorder>degree</zorder>
   <zorder>label_5</zorder>
   <zorder>larrow_b</zorder>
   <zorder>light_b</zorder>
   <zorder>fuel_b</zorder>
   <zorder>fix_b</zorder>
   <zorder>battery_b</zorder>
   <zorder>opendoor_b</zorder>
   <zorder>rarrow_b</zorder>
   <zorder>label_6</zorder>
   <zorder>output</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
